src/declarations/stencil-public-runtime.ts

```diff
@@ -1853,6 +1853,9 @@ export namespace JSXBase {
     onTransitionRunCapture?: (event: TransitionEvent) => void;
     onTransitionStart?: (event: TransitionEvent) => void;
     onTransitionStartCapture?: (event: TransitionEvent) => void;
+
+    // WAI-ARIA Attributes
+    [key: `aria-${string}`]: string | boolean | undefined;
```

Comment:
As these types are mostly relevant for a TSX environment, we should also provide a camel case option.
```suggestion
    [key: `aria-${string}`]: string | boolean | undefined;
    [key: `aria${string}`]: string | boolean | undefined;
```